administrator/language/pl-PL/com_joomlaupdate.ini

```
@@ -78,7 +78,7 @@ COM_JOOMLAUPDATE_UPDATE_LOG_INSTALL="Rozpoczynanie instalacji nowej wersji."
 COM_JOOMLAUPDATE_UPDATE_LOG_START="Aktualizacja rozpoczęta przez użytkownika %2$s (%1$s). Starsza wersja to %3$s."
 COM_JOOMLAUPDATE_UPDATE_LOG_URL="Pobieranie paczki aktualizacyjnej z %s."
 COM_JOOMLAUPDATE_UPDATING_HEAD="Aktualizacja w toku"
-COM_JOOMLAUPDATE_UPDATING_INPROGRESS="Proszę czekać; Joomla jest aktualizowany. To może trochę potrwać."
+COM_JOOMLAUPDATE_UPDATING_INPROGRESS="Proszę czekać; Joomla jest aktualizowana. To może trochę potrwać."
```

Wyraz Joomla! nie ma rodzaju: Polskie kłopoty z odmianą
Nowo narodzony Joomla! sprawiał polskim użytkownikom niemały kłopot płciowo-językowy…. Najpierw, nie wiadomo czemu założono, że to „panienka”, ale w rocznicę narodzin Centrum Projektu oświadczyło, że dziecię płeć ma… męską!, bo jakże inaczej mogła zdecydować męska większość wśród projektantów. Na wskutek tego oświadczenia gdzieniegdzie płeć pozamienialiśmy, ale wątpliwości pozostały.

Tłumacząc Joomla! 1.5, uznaliśmy, że skoro polskie odpowiedniki nazwy - słowa 'razem' czy 'wspólnie' - są przysłówkami, to i nazwę 'Joomla!' należy potraktować jak nieodmienny przysłówek. Ale że kłopot dał o sobie znać w odmianie zaimków i czasowników, zwykliśmy je odnosić do słów ogólniejszych - 'program' czy 'system'. Mówimy zatem i piszemy: „ten Joomla!” a nie „ta Joomla!”.
```suggestion
COM_JOOMLAUPDATE_UPDATING_INPROGRESS="Proszę czekać; Joomla jest aktualizowany. To może trochę potrwać."
```

---

@zwiastunsw Moim zdaniem słowo „Joomla” powinno się odmieniać i mieć rodzaj żeński. Dziwnie się czyta np. „Joomla jest aktualizowany”. Może w wersji angielskiej jest to rodzaj męski, ale w formie polskiej wskazuje na to, że jest to rodzaj żeński, ale nie mnie tu oceniać. Jeśli tak postanowiliście, to tak musi być 😉.